os/customizing-sshd.md

```diff
@@ -43,6 +45,99 @@ coreos:
       Accept=yes
 ```
 
```

Comment:
It would be nice to explain the result of following the example here. E.g.,  "Sshd will now be activated by requests to port #2222, rather than 22", or something like that.